nnoir/nnoir/nnoir.py

```diff
@@ -9,16 +9,7 @@ def __init__(self, message):
 
 
 class NNOIR:
-    def __init__(
-        self,
-        name,
-        generator_name,
-        generator_version,
-        inputs,
-        outputs,
-        values,
-        functions,
-    ):
+    def __init__(self, name, generator_name, generator_version, inputs, outputs, values, functions, description=b""):
```

Comment:
I'm wondering if it is better to  arrange argument as follow instead.(more natural but break compatibility)
`def __init__(self,  name, description, generator_name, generator_version, inputs, outputs, values, functions):`

Reply:
I think the compatibility is more important than the order of arguments.